classes/task/populate_objects_filesize.php

```diff
@@ -43,10 +43,12 @@ public function execute() {
         $maxupdates = !empty($data->maxupdates) ? $data->maxupdates : self::MAX_UPDATES;
 
         // Get all objects without a filesize and join them to a filesize from the files table.
+        // Values less than 0 for object's location indicate an error for the object.
         $sql = "SELECT o.id, o.contenthash, o.timeduplicated, o.location, f.filesize
                   FROM {tool_objectfs_objects} o
              LEFT JOIN {files} f ON o.contenthash = f.contenthash
```

Comment:
shouldn't this be a "Join" instead of a Left Join? - if the point is to get the filesize from the files table, it shouldn't return records when there isn't a files record for the item?